client/src/components/Employees/Employees.js

```diff
@@ -1,13 +1,90 @@
 import React from "react";
 import "./Employees.css";
 import EachEmployee from "./EachEmployee.js";
+import Modal from "react-modal";
+
+function AddEmployee() {
+  function afterOpenModal() {
+    // references are now sync'd and can be accessed.
+    subtitle.style.color = "#black";
+  }
+
+  function openModal() {
+    setIsOpen(true);
+  }
+
+  function closeModal() {
+    setIsOpen(false);
+  }
+
+  const [modalIsOpen, setIsOpen] = React.useState(false);
+
+  var subtitle;
+
+  const customStyles = {
+    content: {
+      top: "50%",
+      left: "50%",
+      right: "auto",
+      bottom: "auto",
+      marginRight: "-50%",
+      height: "45%",
+      width: "32%",
+      transform: "translate(-50%, -50%)"
+    }
+  };
+
+  return (
+    <div>
+      <div className="addbutton">
+        <button onClick={openModal}>+ Add Employees</button>
+      </div>
+      <Modal
```

Comment:
The modal has a horizontal and vertical scroll bar for me. Let's fix that. If you can't see it on your computer, try resizing the window until you do.

Reply:
done